src/css/ams-map.scss

```diff
@@ -0,0 +1,18 @@
+//
+//  AMSTERDAM specific map overrides
```

Comment:
Repo is amsterdam/stijl. Dan lijkt het me logisch dat het specific AMSTERDAM (map) overrides zijn.

Is het misschien passender om leaflet stijling overrides te noemen of iets?

Reply:
verbeterd

Reply:
done